org.jacoco.core.test/src/org/jacoco/core/data/ExecutionDataReaderWriterTest.java

```diff
@@ -142,7 +142,7 @@ public void testInvalidVersion() throws IOException {
 		buffer.write(ExecutionDataWriter.BLOCK_HEADER);
 		buffer.write(0xC0);
 		buffer.write(0xC0);
-		final char version = ExecutionDataWriter.FORMAT_VERSION - 1;
+		final char version = (char) (ExecutionDataWriter.FORMAT_VERSION - 1);
```

Comment:
Interesting that explicit cast not needed in case of inlined value. Learned something new today :satisfied: